datajs/src/parser/csv.js

```diff
@@ -132,3 +129,44 @@ export class Uint8ArrayToStringsTransformer {
     }
   }
 }
+
```

Comment:
Maybe could be good to add a JS DOC for the function.

Reply:
Great idea, will do that!

Reply:
Yea, let's always use JSDoc👍